neuralop/models/fno_block.py

```diff
@@ -94,7 +94,7 @@ def __init__(self, in_channels, out_channels, n_modes,
         if norm is None:
             self.norm = None
         elif norm == 'instance_norm':
-            self.norm = nn.ModuleList([getattr(nn, f'InstanceNorm{self.n_dim}d')(num_features=self.out_channels) for _ in range(n_layers*self.n_norms)])
+            self.norm = nn.ModuleList([getattr(nn, f'InstanceNorm{self.n_dim}d')(num_features=self.out_channels, affine = True) for _ in range(n_layers*self.n_norms)])
```

Comment:
pep8 - no spaces around =, @ashiq24 good to run a checker on your scripts. You can try black but it is quite stringent.